File: 448-FindAllNumbersDisappearedInAnArray/448-FindAllNumbersDisappearedInAnArray.js
// Last updated: 4/14/2026, 12:48:29 PM
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDisappearedNumbers = function(nums) {
    let n = nums.length
    let count = new Array(0)
    let num = new Set(nums)

    for(let i = 1 ; i<n+1 ; i++){
        if (!num.has(i) ){
            count.push(i)
        }
    }return count
};

File: 448-FindAllNumbersDisappearedInAnArray/448-FindAllNumbersDisappearedInAnArray.py
# Last updated: 4/14/2026, 12:48:32 PM
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        num = set(nums)
        n=len(nums)
        count=[]
        for i in range(1,n+1):
            if i not in num:
                count.append(i)
        return count
       
        
    